pandas/_libs/index.pyx

```diff
@@ -606,25 +606,110 @@ cdef class BaseMultiIndexCodesEngine:
                        in zip(self.levels, zip(*target))]
         return self._codes_to_ints(np.array(level_codes, dtype='uint64').T)
 
-    def get_indexer(self, object target, object method=None,
-                    object limit=None):
+    def get_indexer(self, object target) -> np.ndarray:
```

Comment:
I would rename this method, call it get_indexer_no_fill or something to differentiate from get_indexer (where we have elsewhere)

Reply:
done